src/poliastro/twobody/angles.py

```diff
@@ -24,6 +24,126 @@ def _kepler_equation_prime_hyper(F, M, ecc):
     return ecc * np.cosh(F) - 1
 
 
+def _kepler_equation_parabolic(D, M, ecc):
+    return M_parabolic(ecc, D) - M
+
+
+def _kepler_equation_prime_parabolic(D, M, ecc):
+    return M_parabolic_prime(ecc, D)
+
+
+def M_parabolic(ecc, D, tolerance=1e-16):
+    """Computes the Kepler equation r.h.s. in near-parabolic regime
+
+    Parameters
+    ----------
+    D : float
+        Eccentric anomaly (rad).
+    ecc : float
+        Eccentricity,
+    tolerance : float (optional)
+        smallness of the last term in series
+    Returns
+    -------
+    M_parabolic : float
+        kepler equation r.h.s.
+
+    Note
+    -----
+    Taken from the paper <<Robust resolution of Kepler’s equation in all eccentricity regimes>>,
```

Comment:
Could you change the citation style here to something more standard? For example (taken from scholar.google.com, Chicago style):

```
.. [1] Farnocchia, Davide, Davide Bracali Cioci, and Andrea Milani.
   "Robust resolution of Kepler’s equation in all eccentricity regimes."
   Celestial Mechanics and Dynamical Astronomy 116, no. 1 (2013): 21-34.
```

(as explained in https://numpydoc.readthedocs.io/en/latest/format.html#sections, section 13)

Reply:
Also, rename `Note` to `Notes` (and adjust the `-----`).